playbooks/roles/xapi_bridge/defaults/main.yml

```diff
@@ -5,10 +5,12 @@
 
 xapi_bridge_user: xapi
 xapi_bridge_user_group: adm  #perms on tracking.log
+xapi_bridge_log_user: root
```

Comment:
I don't think the Python process will have permission to write to the log files if they are owned by root.

Reply:
OK, we talked about this and it's just for system-level logging.